modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/internal/v2_0/StrictWFS_2_0_Strategy.java

```diff
@@ -662,24 +665,43 @@ protected OperationType getOperationMetadata(final WFSOperationType operation) {
         return ftypeCrss;
     }
 
+    /** Returns parameters defined in OperationsMetadata */
+    @SuppressWarnings("unchecked")
+    private Set<String> findParameters(final String parameterName) {
+        final OperationsMetadataType operationsMetadata = capabilities.getOperationsMetadata();
```

Comment:
I am a bit confused and need to double check the specification; `operations` plural should be the list of operations like `GetCapabilities`, `DescribeFeatureType`, etc... right?

Reply:
When you look for output formats defined globally (for feature content) I assumed you were only going to check the `GetFeature` operation...

Reply:
Checking...

http://docs.opengeospatial.org/is/09-025r2/09-025r2.html#52
  
Common request parameters has Standard presentation parameters including outputFormat parameter so the different operations handle the concept in the same manner. This just describes how outputFormat parameter can be used by the operations; if they have the idea they should use the parameter in a consistent fashion.

It does not imply that each operation support the same output formats (for example GetFeature and DescribeFeatureType provide different information).

* http://docs.opengeospatial.org/is/09-025r2/09-025r2.html#table_12 Table 12 — Parameter domains for WFS operations

Reply:
Okay I see in the example they have:
```
      <ows:Parameter name="version">
         <ows:AllowedValues>
         <ows:Value>2.0.2</ows:Value>
         <ows:Value>2.0.0</ows:Value>
         </ows:AllowedValues>
      </ows:Parameter>
```

So version is a parameter common to all the different operations - you are right!

I have never seen this done for outputFormats; but since one server does it we should support it.